Doc/library/ast.rst

```diff
@@ -1748,43 +1748,55 @@ Type parameters
 :ref:`Type parameters <type-params>` can exist on classes, functions, and type
 aliases.
 
-.. class:: TypeVar(name, bound)
+.. class:: TypeVar(name, bound, default_value)
 
-   A :class:`typing.TypeVar`. ``name`` is the name of the type variable.
-   ``bound`` is the bound or constraints, if any. If ``bound`` is a :class:`Tuple`,
-   it represents constraints; otherwise it represents the bound.
+   A :class:`typing.TypeVar`. *name* is the name of the type variable.
+   *bound* is the bound or constraints, if any. If *bound* is a :class:`Tuple`,
+   it represents constraints; otherwise it represents the bound. *default_value*
+   is the default value, if any.
```

Comment:
It might be worth spelling this out a bit more, since this is now different to what happens at runtime. (I think that's the correct choice; the distinction between `None` for no default and `ast.Constant(value=None)` is quite clean in the AST.)

```suggestion
   is the default value; if the :class:`!TypeVar` has no default, this 
   attribute will be set to ``None``.
```